test/verify/check-machines-create

```diff
@@ -1060,7 +1060,7 @@ class TestMachinesCreate(VirtualMachinesCase):
                 wait(lambda: dialog.name in self.machine.execute("virsh list --persistent"), delay=3)
 
                 # unfinished install script runs indefinitelly, so we need to force it off
-                self.performAction(name, "forceOff")
+                self.test_obj.performAction(name, "forceOff", False)
```

Comment:
That testObj change looks like it should have been part of the previous commit?

Reply:
Ups yes, let's just squash on merge.